samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -91,6 +91,72 @@ To bring the accessory back to its initial state and make provisioning available
 After the factory reset operation, the Fast Pair discoverable advertising gets automatically disabled to prevent a restart of the Fast Pair procedure.
 You need to press a button to set your accessory in the Fast Pair discoverable advertising mode and make it available for the next FMDN provisioning.
 
+.. _fast_pair_locator_tag_fp_adv_policy:
+
+Fast Pair advertising policy
+============================
+
+The Fast Pair advertising is managed by the sample.
+
+The Fast Pair advertising will stay turned on if at least one of the following triggers requested it:
+
+   * UI trigger - On the button action.
+   * FMDN provisioning trigger - On Account Key write operation during FMDN provisioning operation.
+   * Beacon clock synchronization trigger - After the system bootup if the device is provisioned.
+   * DFU mode trigger - On the DFU mode change.
+
+If the Fast Pair advertising is enabled, the sample will automatically select the correct advertising mode:
+
+   * Fast Pair discoverable advertising - without an Account Key stored on the device.
+   * Fast Pair not discoverable advertising - with an Account Key stored on the device.
+
+Removing one of the trigger requests does not automatically mean that the Fast Pair advertising will be disabled, as it requires the removal of all trigger requests.
+It is done to mitigate the situation where the Fast Pair advertising would be disabled by one module but it was required to be enabled by the other module.
+
+The sample automatically disables the advertising by removing the trigger requests right before the factory reset operation.
+
+.. note::
+
+   The Bluetooth advertising is active only until the Fast Pair Provider connects to a Bluetooth Central.
+   After the connection, you can still request turning on the advertising, but it will come into effect only after disconnection.
+
+Device Firmware Update (DFU)
+============================
+
+The locator tag sample uses the :ref:`MCUboot <mcuboot:mcuboot_ncs>` bootloader firmware image that allows you to upgrade the used application firmware image using the :ref:`app_dfu` procedure.
+Over-the-air updates are supported using MCUmgr's Simple Management Protocol (SMP) over Bluetooth.
+
+To enable the DFU functionality use the ``SB_CONFIG_APP_DFU`` sysbuild Kconfig option.
+It is enabled by default if the MCUboot bootloader image is used.
+
```

Comment:
Please write a short section (one-two sentences) on how to change the version of the application (:ref:`zephyr:app-version-details` , `VERSION` file).